client/src/components/IndexPage.js

```diff
@@ -0,0 +1,38 @@
+import React, { useState, useEffect } from "react";
+import { Link } from "react-router-dom";
+
+const IndexPage = (props) => {
+  const [trails, setTrails] = useState([]);
+
+  const getTrails = async () => {
+    try {
+      const response = await fetch("/api/v1/index");
```

Comment:
Think of the longevity of this set up as well - when you then want to fetch one single trail, you would be accessing `api/v1/index/:id` which doesn't read very well what data we're trying to retrieve